profiles/common/modules/features/communities/communities.module

```diff
@@ -51,19 +51,24 @@ function communities_views_pre_render(&$view) {
 function communities_init() {
   drupal_add_css(drupal_get_path('module', 'communities') . '/communities.css');
   global $language;
+  $arguments = explode("/", request_path());
 
   // Set og context on comunities views.
-  if (arg(0) == 'community') {
-    $alias = drupal_get_normal_path("community/" . str_replace(" ", "-", arg(1)), 'en');
+  if ($arguments[0] == 'community') {
+
+    // Sometimes the parameter contains a language suffix, remove it.
+    $suffix_clean = preg_replace('/\\_[^.\\s]{2}$/', '', $arguments[1]);
```

Comment:
`$arguments[0]` will always be set since `request_path()` returns always a string (could be empty, but still a string), we are not sure that `$arguments[1]` is set.